src/client/mixins/lighthouse.ts

```diff
@@ -0,0 +1,46 @@
+import { Page } from 'puppeteer';
+import * as lighthouse from 'lighthouse';
+import * as DesktopConfig from 'lighthouse/lighthouse-core/config/lr-desktop-config';
+import * as MobileConfig from 'lighthouse/lighthouse-core/config/lr-mobile-config';
+
+export class LighthouseAware {
+  public client: Page;
+
+  async getLighthouseScores(url: string, throttleTo: 'desktop' | 'mobile' = 'desktop', categories: string[] = ['performance']) {
+    const browser = this.client.browser();
+    browser.on('targetchanged', async (target) => {
```

Comment:
- [ ] Hmm...  I wonder if any of this in the `browser.on('targetchanged')` handler is necessary?  It looks to be adding some styles that wouldn't matter, since we never display the page.

Reply:
I am trying to remove these now. Actually copy-pasta-ed the guide they have